src/components/Navbar/MobileMenu.tsx

```diff
@@ -20,9 +20,15 @@ import { useAuthWalletContext } from '@/contexts/AuthWalletContext';
 import { SignedInButton } from '../SignedInButton';
 import { ConnectWalletButton } from '../ConnectWalletButton';
 
-import { MENU_ITEMS } from './Navbar';
-
-export const MobileMenu = () => {
+interface MobileMenuProps {
+  menuItems: Array<{
+    name: string;
+    href: string;
+    isExternal?: boolean;
+    isHidden?: boolean;
```

Comment:
maybe we can name it like `requireAuth` or `isProtected`? since it only shows when user is logged in